pygmt/base_plotting.py

```diff
@@ -813,7 +813,7 @@ def legend(self, spec=None, position="JTR+jTR+o0.2c", box="+gwhite+p1p", **kwarg
             lib.call_module("legend", arg_str)
 
     @fmt_docstring
-    @use_alias(R="region", J="projection", B="frame")
+    @use_alias(R="region", J="projection", B="frame", W="pen")
```

Comment:
How should we alias `G`? [I initially thought `color` would be good](https://github.com/GenericMappingTools/pygmt/pull/480#discussion_r442518040), but GMT.jl uses `fill` (see https://www.generic-mapping-tools.org/GMT.jl/latest/#GMT.text). Maybe `fill` then, since color could be confused with the text color?

Reply:
`fill` is good to me.

Reply:
Fill is much better. (I have been lurking here.)

Reply:
Cool, feel free to suggest any other changes :wink: